launch.go

```diff
@@ -593,6 +593,21 @@ func main() {
 			return cli.ShowAppHelp(c)
 		}
 
+		if len(token) == 0 {
+			log.Println("Error token is not passed.")
+			os.Exit(0)
+		}
+
+		if len(url) == 0 {
+			log.Println("Error API uri is not specified.")
+			os.Exit(0)
+		}
+
+		if len(storeURL) == 0 {
+			log.Println("Error store uri is not specified.")
```

Comment:
Don't we have to update build status?

Reply:
I fix it to use `exit()`.

Reply:
Sorry, It seems that the validations for `url` and `storeUrl` are needless.
These has static default value. So I' remove these validations.
https://github.com/screwdriver-cd/launcher/blob/68a22686724fb00fa40b9024f72dc390765715da/launch.go#L535